flink-cdc-connect/flink-cdc-source-connectors/flink-connector-debezium/src/main/java/org/apache/flink/cdc/debezium/internal/DebeziumChangeFetcher.java

```diff
@@ -268,8 +268,9 @@ private void emitRecordsUnderCheckpointLock(
         synchronized (checkpointLock) {
             T record;
             while ((record = records.poll()) != null) {
+                // If the snapshot does not end or no latest data is entered, -1 is reported
                 emitDelay =
-                        isInDbSnapshotPhase ? 0L : System.currentTimeMillis() - messageTimestamp;
+                        isInDbSnapshotPhase ? -1L : System.currentTimeMillis() - messageTimestamp;
```

Comment:
Maybe we can verify this change by adding a test case similar to `MySqlSourceITCase#testSourceMetrics`